awsguard-typescript/index.ts

```diff
@@ -0,0 +1,3 @@
+import { AwsGuard } from "@pulumi/awsguard";
+
+new AwsGuard({ all: "advisory" });
```

Comment:
This could have been simply:

```typescript
new AwsGuard();
```

But I decided to show off passing the args, to demonstrate how it can be configured.